Fix the following error:

    ./Source/MediaInfo/MediaInfo_Internal.cpp: In function 'void MediaInfoLib::SignalHandler(int)':
    ./Source/MediaInfo/MediaInfo_Internal.cpp:979:21: error: 'SIGINT' was not declared in this scope
      979 |     if (SignalType==SIGINT)
          |                     ^~~~~~

--- a/Source/MediaInfo/MediaInfo_Internal.cpp
+++ b/Source/MediaInfo/MediaInfo_Internal.cpp
@@ -65,6 +65,7 @@
 #endif //MEDIAINFO_ADVANCED
 #if MEDIAINFO_ADVANCED && defined(MEDIAINFO_FILE_YES)
     #include <limits>
+    #include <signal.h>
     #ifdef WINDOWS
     namespace WindowsNamespace
     {
@@ -75,7 +76,6 @@
     #elif defined(_POSIX_PRIORITY_SCHEDULING)
         #include <sched.h>
         #include <unistd.h>
-        #include <signal.h>
     #endif //_POSIX_PRIORITY_SCHEDULING
     #include <ctime>
 #endif
